docs/source/gbd2017_models/risk_attributable_causes/iron_deficiency/index.rst

```diff
@@ -473,12 +521,15 @@ Other Model Notes/Strategies
 Neonatal Age Groups
 ^^^^^^^^^^^^^^^^^^^
 
+Neonatal age groups should be excluded from the process described in this 
```

Comment:
I think you said at one point this is due to lack of data from gbd? i think it would be helpful to include the reasoning here.